src/adapter/http/src/simple_protocol/handlers.rs

```diff
@@ -279,8 +279,15 @@ fn get_base_dataset_url(
 /////////////////////////////////////////////////////////////////////////////////
 
 fn get_api_server_url(host: &str) -> Url {
-    let scheme = std::env::var("KAMU_PROTOCOL_SCHEME").unwrap_or_else(|_| String::from("http"));
-    Url::parse(&format!("{scheme}://{host}")).unwrap()
+    // TODO: Use value from config not envvar
+    //       https://github.com/kamu-data/kamu-node/issues/45
+    let raw_base_url = std::env::var("KAMU_BASE_URL").unwrap_or_else(|_| {
+        let scheme = std::env::var("KAMU_PROTOCOL_SCHEME").unwrap_or_else(|_| String::from("http"));
```

Comment:
If we introduce this base url, maybe we don't need scheme. I think it is safe to assume that we either have a deployment where it is defined, or use HTTP via localhost.

Reply:
Done